File: notification-service/src/test/java/com/eirsteir/coffeewithme/notification/NotificationServiceApplicationTests.java
package com.eirsteir.coffeewithme.notification;

import com.eirsteir.coffeewithme.notification.config.EventuateTestConfig;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest(classes = {NotificationServiceApplication.class, EventuateTestConfig.class})
class NotificationServiceApplicationTests {

    @Test
    void contextLoads() {
    }

}

File: notification-service/src/test/kotlin/com/eirsteir/coffeewithme/notification/NotificationServiceApplicationTests.kt
package com.eirsteir.coffeewithme.notification

import com.eirsteir.coffeewithme.notification.config.EventuateTestConfig
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles

@ActiveProfiles("test")
@SpringBootTest(classes = [NotificationServiceApplication::class, EventuateTestConfig::class])
internal class NotificationServiceApplicationTests {
    @Test
    fun contextLoads() {
    }
}